sympy/integrals/manualintegrate.py

```diff
@@ -725,10 +725,18 @@ def trig_cotcsc_rule(integral):
                 cotcsc_csceven_condition: cotcsc_csceven
             })((a, b, m, n, integrand, symbol))
 
+def trig_sin2cos_rule(integral):
+    integrand, symbol = integral
+    a = sympy.Wild('a', exclude=[1/(sympy.sin(2*symbol)*sympy.cos(symbol))])
+    match = integrand.match(sympy.sin(2*symbol)*sympy.cos(symbol)*a)
```

Comment:
At first I'm thinking this rule only match cos(x)*sin(2x), however after you ask about it, I don't know why I include cos(symbol) in here cause it seems to be unnecessary. Maybe I was thinking matching sin(2x) only will sin(2x) convert to 2sin(x)cos(x) but this is not true since trig_rule would dealt with pure sin function. Let me change this